packages/design-system/src/components/Table/Table.jsx

```diff
@@ -1,8 +1,7 @@
-import React, { useEffect, useRef, useState } from 'react';
 import Alert from '../Alert/Alert';
 import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
One thing that might be nice to include is a comment linking to this PR, this way we can revert these changes when we eventually get everyone onto react 16.8